src/model/AbstractRepository.ts

```diff
@@ -0,0 +1,46 @@
+import { mapping } from "cassandra-driver";
+
+export abstract class AbstractRepository<T extends { [key: string]: any }, U> {
```

Comment:
`{ [key: string]: any }` Why a generic dictionary definition and not a simple Type name?

Reply:
Beacause `mapper.insert()` demands a Key-Value Pair. I'll try to make this a little less verbose.